props/blaster.h

```diff
@@ -112,38 +113,43 @@ class Blaster : public PROP_INHERIT_PREFIX PropBase {
     return random <= percent ? true : false;
   }
 
+  virtual void CheckEmpty() {
+    if (max_shots_ != -1 && shots_fired_ >= max_shots_) {
+      empty_ = true;
+      SaberBase::DoEffect(EFFECT_EMPTY, 0);
+      return;
```

Comment:
what's the point of having a "return" here?

Reply:
So we don't continue on and fire anyway if we're in MODE_AUTO?